pyblish_qml/ipc/client.py

```diff
@@ -105,6 +113,10 @@ def _listen():
                     elif response.get("header") == "pyblish-qml:popen.parent":
                         self.channels["parent"].put(line)
 
+                    elif response.get("header") == "pyblish-qml:server.pulse":
+                        self._kill.cancel()  # reset timer
```

Comment:
Ahaa. :) Nice one.

Is `30` seconds or milli-seconds? Ms might be a little short, in case a pulse is delayed for some reason.

Reply:
Opss, it 30 second.

The server will send pulse every 15 second, and the client here will reset every 30 second.
I'll append doc string about this to `_self_destruct` in next commit :P